src/components/Payment/index.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import creditcard from '../../assets/creditcard.png'
+import date from '../../assets/date.png'
+import coin from '../../assets/coins.png'
+import locked from '../../assets/locked.png'
+
+import {
+  Input
+} from 'reactstrap';
+
+
```

Comment:
Codacy found an issue: [More than 1 blank line not allowed.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5714105)